lib/main.go

```diff
@@ -200,6 +200,16 @@ func (obj *Main) Exit(err error) {
 	obj.exit <- err // trigger an exit!
 }
 
+func (obj *Main) createTmpPrefix(prefix *string, hostname string) error {
+	var err error
+	if *prefix, err = ioutil.TempDir("", obj.Program+"-"+hostname+"-"); err != nil {
+		return err
+	}
+	log.Println("Main: Warning: Working prefix directory is temporary!")
```

Comment:
logging should be in main, not buried in a helper function please

Reply:
I was thinking to pull the entire prefix thing into a separate function for easier testing anyways, functionally the helper is really trivial in that it might be better to be less DRY here.